CI/Dockerfile

```diff
@@ -33,64 +61,208 @@ RUN apt-get -y update; \
+    cd build && \
+    # static building of moab is also an option and was required for older versions
+    # cmake ../moab -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_install_dir} \
+    #               -DCMAKE_INSTALL_PREFIX=${moab_install_dir} \
+    #               -DCMAKE_CXX_COMPILER=${CXX} \
+    #               -DCMAKE_C_COMPILER=${CC} \
+    #               -DENABLE_BLASLAPACK=OFF \
+    #               -DBUILD_SHARED_LIBS=OFF \
+    #               -DENABLE_FORTRAN=OFF && \
+    # make -j${ci_jobs} && \
+    # make install && \
+    # rm -rf * && \
+    cmake ../moab -DCMAKE_INSTALL_RPATH=${hdf5_install_dir}/lib:${moab_install_dir}/lib \
+                  -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_install_dir} \
+                  -DCMAKE_INSTALL_PREFIX=${moab_install_dir} \
+                  -DCMAKE_CXX_COMPILER=${CXX} \
+                  -DCMAKE_C_COMPILER=${CC} \
+                  -DENABLE_BLASLAPACK=OFF \
+                  -DBUILD_SHARED_LIBS=ON \
+                  -DENABLE_FORTRAN=OFF \
+                  -DENABLE_PYMOAB=ON && \
+    mkdir -p ${moab_build_dir}/build/pymoab/lib/python3.6/site-packages && \
+    PYTHONPATH=${moab_build_dir}/build/pymoab/lib/python3.6/site-packages:${PYTHONPATH} make -j${ci_jobs} && \
+    PYTHONPATH=${moab_build_dir}/build/pymoab/lib/python3.6/site-packages:${PYTHONPATH} make install && \
+    cd && \
+    rm -rf ${moab_build_dir}
+
+ENV double_down_build_dir=${build_dir}/double-down
+ENV double_down_install_dir=${install_dir}/double-down
+
+# Clone and install Double-Down
+# performed after moab install as double-down requires moab
+RUN mkdir -p ${double_down_build_dir}/build && \
+    cd ${double_down_build_dir} && \
+    git clone --shallow-submodules --single-branch --branch v1.0.0 --depth 1 https://github.com/pshriwise/double-down.git && \
+    cd build && \
+    cmake ../double-down -DMOAB_DIR=${moab_install_dir} \
+                         -DCMAKE_INSTALL_PREFIX=${double_down_install_dir} \
+                         -DEMBREE_DIR=${embree_install_dir} && \
+    make -j${ci_jobs} && \
+    make -j${ci_jobs} install
+
+
+FROM moab as dagmc
+
+# accessing gloabl ARGs in build stage
+ARG build_mw_reg_tests
+ARG install_dir
+ARG build_dir
+ARG static_exe
+ARG CXX
+ARG CC
+
+ENV dagmc_build_dir=${build_dir}/dagmc
+ENV dagmc_install_dir=${install_dir}/dagmc
+
+# copies the enitre git repo into the dockerfile to ensure the submodules can also be found
+RUN mkdir -p ${dagmc_build_dir}/build
+COPY . ${dagmc_build_dir}
+RUN cd  ${dagmc_build_dir} && git submodule update --init
+
+RUN mkdir -p ${dagmc_build_dir}/build && \
+    cd ${dagmc_build_dir} && \
+    cd build && \
+    cmake ../${dagmc} -DMOAB_DIR=${moab_install_dir} \
+                      -DBUILD_GEANT4=ON \
+                      -DGEANT4_DIR=${geant4_install_dir} \
+                      -DBUILD_CI_TESTS=ON \
+                      -DBUILD_MW_REG_TESTS=${build_mw_reg_tests} \
```

Comment:
this line could be changed

Reply:
Discussed this at the DAGMC meeting yesterday and this is the preferred route, I've opened issue #858 to track the missing tar.gz file with the models in it.